module/localisation/BallLocalisation/src/BallModel.hpp

```diff
@@ -73,7 +74,8 @@ namespace module::localisation {
 
             const Eigen::Matrix<Scalar, 3, 1> rBWw(state[PX], state[PY], field.ball_radius);
             const Eigen::Matrix<Scalar, 3, 1> rBCc_cart(Eigen::Affine3d(Hcw) * rBWw);
-            return cartesianToSpherical(rBCc_cart);
+
+            return inverseDistanceCartesianToSpherical(rBCc_cart);
```

Comment:
Do we need to call this `_cart`?
```suggestion
            return inverseDistanceCartesianToSpherical(rBCc);
```